.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+---
+
+<!-- Text above this line will be added to the commit once the PR is merged -->
+
+### Pull Request checklist
+
+<!-- Before submitting the PR, please address each item -->
+
+- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/RecordReplay/devtools/blob/master/CHANGELOG.md) or does not need one
+- [ ] **Screenshot**: This PR includes screenshots of the old and new visual changes
```

Comment:
Small nit - not all PRs have a corresponding visual change. Should we address that specifically?